src/components/Button/index.tsx

```diff
@@ -524,15 +525,17 @@ const BaseThemeButton = styled.button<BaseThemeButtonProps>`
 `
 
 interface ThemeButtonProps extends React.ComponentPropsWithoutRef<'button'>, BaseThemeButtonProps {}
+type ThemeButtonRef = HTMLButtonElement
 
-export const ThemeButton = ({ children, ...rest }: ThemeButtonProps) => {
+export const ThemeButton = forwardRef<ThemeButtonRef, ThemeButtonProps>(({ children, ...rest }, ref) => {
   return (
-    <BaseThemeButton {...rest}>
+    <BaseThemeButton {...rest} ref={ref}>
       <ButtonOverlay />
       {children}
     </BaseThemeButton>
   )
-}
+})
+ThemeButton.displayName = 'ThemeButton'
```

Comment:
in general react is deprecating this and encouraging just `function ThemeButton` on the inner function, but idk if our eslint maybe doesnt like that

Reply:
Yeah I never use displayName, but eslint complains about this when ref forwarding.. 🤔